config/bidderinfo.go

```diff
@@ -124,6 +124,9 @@ type Syncer struct {
 
 	// SupportCORS identifies if CORS is supported for the user syncing endpoints.
 	SupportCORS *bool `yaml:"supportCors" mapstructure:"support_cors"`
+
+	// ForceSyncType allows a bidder to override their callback type "b" for iframe, "i" for redirect
+	ForceSyncType string `yaml:"forceSyncType" mapstructure:"force_sync_type"`
```

Comment:
This is called `formatOverride` in PBS-Java. Perhaps we can use that same name for PBS-Go. I also recommend to make a slight change to the description:

> ForceSyncType allows a bidder to override their callback type.  Valid values are "b" for iframe or "i" for redirect.